gridstatus/pjm.py

```diff
@@ -34,387 +38,10 @@ class PJM(ISOBase):
-        "1552843818",
-        "57967665",
-        "1552843913",
-        "1552843915",
-        "1552843916",
-        "1356162213",
-        "1356162214",
-        "50401",
-        "48934161",
-        "48934163",
-        "48934165",
-        "48934167",
-        "48934169",
-        "36181299",
-        "50488",
-        "50489",
-        "50490",
-        "36181325",
-        "2156113262",
-        "50542",
-        "50543",
-        "50557",
-        "50558",
-        "2156113284",
-        "50578",
-        "50579",
-        "50581",
-        "50621",
-        "50622",
-        "87901631",
-        "50628",
-        "50629",
-        "50654",
-        "50655",
-        "50659",
-        "50660",
-        "50661",
-        "50662",
-        "2156111333",
-        "1048047",
-        "1048049",
-        "1048050",
-        "1048051",
-        "1048052",
-        "21601782",
-        "21601783",
-        "21601784",
-        "21601785",
-        "21601786",
-        "50695",
-        "50696",
-        "50697",
-        "50698",
-        "50699",
-        "2041990671",
-        "123901459",
-        "123901461",
-        "123901463",
-        "123901465",
-        "123901467",
-        "50715",
-        "50716",
-        "50717",
-        "50727",
-        "50728",
-        "50729",
-        "50730",
-        "2156113457",
-        "2156113469",
-        "50764",
-        "2156113488",
-        "50769",
-        "50770",
-        "50771",
-        "50777",
-        "50778",
-        "50779",
-        "123901537",
-        "123901539",
-        "123901543",
-        "31020649",
-        "123901545",
-        "31020651",
-        "31020653",
-        "50809",
-        "50810",
-        "50811",
-        "50812",
-        "50813",
-        "50814",
-        "50817",
-        "50818",
-        "1165479564",
-        "2156109456",
-        "50887",
-        "50888",
-        "50893",
-        "50894",
-        "50911",
-        "50915",
-        "32417525",
-        "32417527",
-        "2156111608",
-        "1218914041",
-        "1218914042",
-        "1218914043",
-        "32417545",
-        "32417547",
-        "1183231801",
-        "32417599",
-        "32417601",
-        "32417603",
-        "32417605",
-        "51019",
-        "51020",
-        "51021",
-        "1348263767",
-        "32417625",
-        "32417627",
-        "32417629",
-        "32417631",
-        "32417633",
-        "32417635",
-        "1379268471",
-        "1379268472",
-        "1379268473",
-        "1379268474",
-        "1379268475",
-        "1379268476",
-        "63381383",
-        "63381385",
-        "2156111770",
-        "2156109760",
-        "2156109763",
-        "2156109765",
-        "2156109768",
-        "2156109772",
-        "2156109777",
-        "5021665",
-        "5021666",
-        "5021667",
-        "2156111847",
-        "93353961",
-        "93353963",
-        "93353965",
-    ]
-
+    location_types = LOCATION_TYPES
```

Comment:
Since these constants are mutable types, if we were to make a change to these attributes in this class, that would change the constant itself. It's safer to make copies of these constants:

```python
location_types = LOCATION_TYPES[:]
hub_node_ids = HUB_NODE_IDS[:]
...
```

Reply:
Thanks, that seems like a good solution.